packages/blade/src/components/Text/Decisions/Decisions.md

```diff
@@ -0,0 +1,87 @@
+The text component is categorised into multiple parts
+
+- BaseText - Internal Component, not published to consumers
+- Title - Public Component
+- Heading - Public Component
+- Text - Public Component
+- Code - Public Component
+
+Here's how the hierarchy will looks like
+![Component Breakdown](./component-breakdown.png)
+
+The APIs for all the above component will look something like below
+
+- `BaseText`
+
+  This component is just meant to be used for internal purposes for Blade
+  Here's how the API will look like
+
+  ```jsx
+  <BaseText
+    color="surface.text | feedback.text | button.text"
+    fontFamily="fontfamily.name"
+    fontSize="fontSize.name"
+    fontWeight="fontWeight.name"
+    lineWeight="lineWeight"
+    letterSpacing="letterSpacing"
+    formatting="bold/italic/strike"
+    as="h1-h6/p/span"
+  />
```

Comment:
Can we also include `textAlign`? There are many use-cases of it.

Reply:
can you share some example?